src/python/WMCore/Cache/GenericDataCache.py

```diff
@@ -3,45 +3,47 @@
 import traceback
 import logging
 
+
 class MemoryCacheStruct(object):
     """
     WARNING: This can be used in multi thread by registering on GenericDataCache
     But this cache is not thread saft.
     """
-    def __init__(self, expire, func, kwargs=None):
+
+    def __init__(self, expire, func, initCacheValue=None, kwargs=None):
         """
-        expire is the seconds which cache will be refreshed when cache is order than the expire.
-        func is the fuction which cache data is retrived
+        expire is the seconds which cache will be refreshed when cache is older than the expire.
+        func is the fuction which cache data is retrieved
         kwargs are func arguments for cache data
         """
-        self.data = None
+        self.data = initCacheValue
```

Comment:
Yep, that's what I wanted and that's why I explicitly set it here:
https://github.com/dmwm/WMCore/pull/7593/files#diff-8468e9d008c1cdfe50bf6725d0eac87bR215